LockTests/Models/UserSpec.swift

```diff
@@ -110,6 +110,12 @@ class UserSpec: QuickSpec {
                 user.reset()
                 expect(user.additionalAttributes["first_name"]).to(beNil())
             }
+            
+            it("should clear root attributes") {
+                user.rootAttributes["family_name"] = "Doe"
+                user.reset()
+                expect(user.rootAttributes["first_name"]).to(beNil())
```

Comment:
Is `first_name` a typo here, or should it be testing that `family_name` is Nil? If not, the intent of the test isn't too clear.

Reply:
I think the intent is that he sets a value first, then clears the map, and then checks that the map no longer has that value he first added. So yes, Steve is right and it should be checking against the key added before.

```suggestion
                expect(user.rootAttributes["family_name"]).to(beNil())
```

Reply:
Yes, was a typo